src/cow-react/constants/routes.ts

```diff
@@ -1,10 +1,7 @@
-export const LIMIT_ORDERS_PATH = 'limit-orders'
-
 // ENUM with routes
 export enum Routes {
   HOME = '/',
-  SWAP = '/swap',
-  SWAP_OUTPUT_CURRENCY = '/swap/:outputCurrency',
+  SWAP = '/:chainId?/swap/:inputCurrencyId?/:outputCurrencyId?',
```

Comment:
Can we add a TODO or a GH issue to update our docs when this change is released?

Reply:
https://docs.google.com/document/d/1P9I4IkfsNXjyhzahhLGequaNKoGik0jP9lTO4TIBNfw/edit#heading=h.1ki9rf7ea5j5
We have a task here